sanicbot/core/utils.py

```diff
@@ -0,0 +1,17 @@
+import aiofiles as aiofiles
+from discord import Embed
+from discord.ext.commands import Context
+
+
+async def failure_message(ctx: Context, message: str):
+    await ctx.send(':x: **' + message + '**')
+
+
+async def success_message(ctx: Context, message: str, embed: Embed = None):
+    message = ':white_check_mark: ** ' + message + '**'
+    await ctx.send(message, embed=embed)
+
+
+async def read_file(path, as_array=False):
+    async with aiofiles.open('resources/' + path) as f:
+        return [line.strip() async for line in f] if as_array else await f.read()
```

Comment:
1. Use `readlines` instead of reading line by line.
2. I think you should get rid of `aiofiles`. It seems unnecessary.

Something like this is much cleaner and easier to read:

```python
        if as_array:
            lines = await f.readlines()
            return [line.strip() for line in lines]
        else:
            return await f.read()
```

However, since there are multiple return types, you should annotate the function accordingly.